movie.rb

```diff
@@ -0,0 +1,19 @@
+require './item'
+
+class Movie < Item
+  attr_accessor :silent
+
+  def initialize(publish_date, silent)
+    super(publish_date)
+    @silent = silent
+  end
+
+  def can_be_archived?
+    parent = super()
+    return true if parent || silent
```

Comment:
- I would suggest you stick to code the way it was from the beginning, on line 13 your method should return true if parent | | @silent and not silent
Kindly check that and make changes